x-pack/plugins/actions/server/lib/action_executor.ts

```diff
@@ -139,9 +148,19 @@ export class ActionExecutor {
         const actionLabel = `${actionTypeId}:${actionId}: ${name}`;
         logger.debug(`executing action ${actionLabel}`);
 
+        const task = taskInfo
+          ? {
+              task: {
+                scheduled: taskInfo.scheduled.toISOString(),
+                schedule_delay: Millis2Nanos * (Date.now() - taskInfo.scheduled.getTime()),
+              },
```

Comment:
heh, ya, didn't get around to it, but we do have the top-level `task` object we can provide more goodies for later.

One of the issues with adding the task manager SO reference, is that we'd also have to indicate the index, since it's not in `.kibana` - or I guess that's my understanding, that you'd need to know the index name to actually look them up.  Or we could special case some of the "types" I suppose, if the existing SO client doesn't magically find them.

Reply:
>One of the issues with adding the task manager SO reference, is that we'd also have to indicate the index

Reply:
>One of the issues with adding the task manager SO reference, is that we'd also have to indicate the index

I think this would be ok as long as the task SO is accessed via the saved objects client/repository. The SO repository would automatically know it's a type of `task` and look up the index `task` belongs into. All good to defer this 👍